codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -108,8 +108,15 @@ public void channelHandlerContext(ChannelHandlerContext ctx) {
         this.ctx = ctx;
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * This method must be called from the same event loop context as the {@link ChannelHandlerContext}
```

Comment:
@normanmaurer @Scottmitch what are your thoughts about using the `@NotThreadSafe` annotation for classes expected to be called within the event loop?